samples/RhythmGame/src/main/cpp/audio/RenderableAudio.h

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  */
 
-#ifndef RHYTHMGAME_AUDIORENDERER_H
-#define RHYTHMGAME_AUDIORENDERER_H
+#ifndef RHYTHMGAME_RENDERABLEAUDIO_H
+#define RHYTHMGAME_RENDERABLEAUDIO_H
 
 
 #include <cstdint>
 
 class RenderableAudio {
 
 public:
+    virtual ~RenderableAudio(){};
```

Comment:
nit: It is recommended to put "= default" instead of using empty body.

Reply:
Done. nits = better code 👍